operate/webapp/src/main/java/io/camunda/operate/webapp/security/sso/TokenAuthentication.java

```diff
@@ -44,8 +44,14 @@ public class TokenAuthentication extends AbstractAuthenticationToken {
 
   public static final String ORGANIZATION_ID = "id";
   public static final String ROLES_KEY = "roles";
-  private static Logger logger = LoggerFactory.getLogger(TokenAuthentication.class);
-  @JsonIgnore private final Integer lock = 0;
+  private static final Logger LOGGER = LoggerFactory.getLogger(TokenAuthentication.class);
+
+  // Need to research further whether a lock shared between all instances is the desired behavior.
+  // This was originally an Integer member variable (non-static) but due to integer pooling, it
+  // was effectively a shared instance. This static object mirrors that behavior to keep consistent
+  // behavior until we research further.
+  @JsonIgnore private static final Object RESOURCE_PERMISSIONS_LOCK = new Object();
```

Comment:
I think se don't really need `@JsonIgnore` for the static field? Or should we leave it for the case when we fix the problem discussed in comment? 🤔